web/types/next-auth.d.ts

```diff
@@ -32,11 +34,18 @@ declare module "next-auth" {
     image?: PrismaUser["image"];
     admin?: PrismaUser["admin"];
     emailVerified?: string | null; // iso datetime string, need to stringify as JWT & useSession do not support Date objects
-    projects: {
-      id: PrismaProject["id"];
-      name: PrismaProject["name"];
-      role: PrismaMembership["role"];
-      cloudConfig: z.infer<typeof cloudConfigSchema> | null;
+    canCreateOrganizations: boolean; // default true, allowlist can be set via LANGFUSE_ALLOWED_ORGANIZATION_CREATORS
```

Comment:
what is this? I just searched the project for the env variable and this property and did not find anything

Reply:
see env.mjs and .env.prod.example, implemented in orgRouter and buttons in frontend